setup.py

```diff
@@ -0,0 +1,37 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="cyanobyte",
+    version="0.0.1",
+    #author="",
+    #author_email="",
+    description="A package that generates library files for a peripheral given an intermediary layer (YAML files)",
+    long_description=long_description,
+    long_description_content_type=" text/markdown",
+    url="https://github.com/google/cyanobyte",
+    packages=setuptools.find_packages(exclude=("test",)),
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: OS Independent",
+    ],
+    python_requires=">=3.0",
+    install_requires=[
```

Comment:
Does this list intend to have versions as well? Is it possible to pull from requirements.txt directly for maintainability?